splunk_add_on_ucc_framework/utils.py

```diff
@@ -42,6 +43,21 @@ def get_license_path(file_name: str) -> str:
     return join(dirname(__file__), "templates", "Licenses", f"{file_name}.txt")
 
 
+def check_author_name(source: str, app_manifest: AppManifest) -> None:
+    check_path = join(source, "default", "app.conf")
+    if isfile(check_path):
+        app_conf = conf_parser.TABConfigParser()
+        app_conf.read(check_path)
+        app_conf_content = app_conf.item_dict()
+        if (
+            app_manifest.get_authors()[0]["name"]
+            != app_conf_content["launcher"]["author"]
+        ):
+            logger.warning(
```

Comment:
Let's recommend where a developer should put a correct value (`app.manifest` or `app.conf`).

Reply:
Updated logger statement.